__tests__/components/HeaderTitle.test..js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+import HeaderTitle from '../../src/components/header/HeaderTitle';
+import React from 'react';
+
+it('renders correctly', () => {
+    const tree = renderer
+        .create(<HeaderTitle title="Testing Title" />)
+        .toJSON();
+    expect(tree).toMatchSnapshot();
+});
```

Comment:
to benefit from snapshot tests you must commit snapshots that they produce. The only goal of snapshot tests is to be sure that your changes didn't affect html if it didn't suppose to